test/test_kubeclient.rb

```diff
@@ -38,6 +38,28 @@ def test_no_version_passed
     assert_equal 'http://localhost:8080/api/v1', rest_client.url.to_s
   end
 
+  def test_pass_proxy
+    uri = URI::HTTP.build(port: 8080)
+    uri.hostname = 'localhost'
```

Comment:
please note that here the "hostname" property is used when building the uri, but on line 44, the "host" property is used for building another uri. can you explain why?